Update the quickstart guide - Closes #1101 (#1103)
LiskHQ/lisk-docs: modules/ROOT/pages/quickstart.adoc

   create src/commands/blockchain/hash.ts
   create src/commands/blockchain/import.ts
   create src/commands/blockchain/reset.ts
   create src/commands/config/create.ts
   create src/commands/config/show.ts
   create src/commands/forger-info/export.ts
   create src/commands/forger-info/import.ts
   create src/commands/forging/config.ts
   create src/commands/forging/disable.ts
   create src/commands/forging/enable.ts
   create src/commands/forging/status.ts
   create src/commands/genesis-block/create.ts
   create src/commands/node/info.ts
   create src/commands/passphrase/decrypt.ts
   create src/commands/passphrase/encrypt.ts
   create src/commands/transaction/create.ts
   create src/commands/transaction/get.ts
   create src/commands/transaction/send.ts
   create src/commands/transaction/sign.ts
   create test/commands/account/create.spec.ts
   create test/unit/modules/.gitkeep
----

NOTE: The application CLI offers various commands to conveniently manage your blockchain application.

As shown in the snippet above, a new folder `src/commands/` will be created, which contains all files for the available command-line commands of the application.

> To see an overview about all available CLI commands, execute the following in the root folder of the application:

Reviewer: ```suggestion
To see an overview of all the available CLI commands, execute the following in the root folder of the application:
```

./new_app/
[source,bash]
----
./bin/run
----

== Start the application

> Use the following command to start the blockchain application.

Reviewer: ```suggestion
Execute the following command to start the blockchain application.
```

./new_app/
[source,bash]
----
./bin/run start
----

> To verify the application start, check the log messages in the terminal.

Reviewer: ```suggestion
To verify that the application has started, check the log messages in the terminal.
```

If the start was successful, the application will enable forging for all genesis delegates and will start to add new blocks to the blockchain every 10 seconds.

//TODO: Create updated gif with the new command
//image:node-start.gif[]

[NOTE]
====
After completing these steps, the default blockchain application of the Lisk SDK will now be running.

It is now possible to customize your application by registering new modules and plugins, and also adjusting the genesis block and config to suit your specific use case.

> See more information in th following guides:

Reviewer: ```suggestion
More information can be found in the following guides:
```
